api/src/form_schema/forms/sf424.py

```diff
@@ -240,29 +232,29 @@
         },
         "fax": {
             "allOf": [{"$ref": "#/$defs/phone_number_field"}],
-            "title": "fax Number",
+            "title": "Fax Number",
             "description": "Enter the fax Number.",
         },
         "email": {
             "type": "string",
-            "title": "email",
+            "title": "Email",
             "description": "Enter a valid email Address.",
             "format": "email",
         },
         "applicant_type_code": {
             # NOTE: In the xml model, this is 3 separate fields, we joined them together
             # into a single value.
             "type": "array",
-            "title": "Type of applicant",
+            "title": "Type of Applicant",
             "description": "Select the appropriate applicant types.",
             "minItems": 1,
             "maxItems": 3,
             "items": {
                 "type": "string",
                 "enum": [
-                    "A: state Government",
-                    "B: county Government",
-                    "C: city or Township Government",
+                    "A: State Government",
+                    "B: County Government",
+                    "C: City or Township Government",
```

Comment:
Changing this is correct (this was actually a mistake I made when renaming things), but you'll also need to fix the tests to use the correct values - just update the strings in the test setup in these two places:
https://github.com/HHS/simpler-grants-gov/blob/main/api/tests/src/form_schema/forms/test_sf424.py#L208
https://github.com/HHS/simpler-grants-gov/blob/main/api/tests/src/form_schema/forms/test_sf424.py#L228